dashboard/modules/job/tests/test_job_manager.py

```diff
@@ -848,7 +848,7 @@ def handler(*args):
         check_job_stopped,
         job_manager=job_manager,
         job_id=job_id,
-        timeout=JobSupervisor.WAIT_FOR_JOB_TERMINATION_S,
+        timeout=JobSupervisor.WAIT_FOR_JOB_TERMINATION_S + 1,
```

Comment:
I would suggest waiting for much longer than this, maybe `WAIT_FOR_JOB_TERMINATION_S + 10`. Sometimes the CI machines are _very_ slow.